lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -625,9 +635,15 @@ def write(chunk)
                                             severity,
                                             ts_secs,
                                             ts_nanos)
-
           trace = record.delete(@trace_key)
-          entry.trace = trace if trace
+          if trace
```

Comment:
We probably wanna check against empty strings as well? I'm leaning towards not formatting the field if it comes as an empty string. WDYT?

BTW, Ruby will take if <empty_string> as `true`.

```
$ irb
2.4.3 :001 > empty_string = ""
 => ""
2.4.3 :002 > if empty_string then true else false end
 => true
```

Reply:
https://gist.github.com/jfarmer/2647362

Reply:
Fun read. Oh Ruby... shoot me:

```
$ irb
2.4.3 :001 > if 0 then true else false end
 => true
```